stylesheet.css

```diff
@@ -0,0 +1,240 @@
+body {
+  background-color: #f8f8f8;
+  margin: 0;
+}
+
+.navbar {
+  display: flex;
+  background-color: white;
+  height: 56px;
+  padding: 0 16px;
+  align-items: center;
+  margin: 0;
+}
+
+.menu {
+  margin: 0 8px 0 0;
+  width: 35px;
+  height: 30px;
+  padding: 8px;
+  box-sizing: border-box;
+  display: inline-block;
+  position: relative;
+}
+
+.menu_button {
+  vertical-align: middle;
+  color: inherit;
+  outline: none;
+  background: none;
+  margin: 0;
+  border: none;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+  line-height: 0;
+  cursor: pointer;
+}
+
+.logo {
+  padding: 8px;
+  margin: 0 100px 0 0;
+  vertical-align: middle;
+}
+
+.menu_button img {
+  width: 100%;
+  height: 100%;
+}
+
+.search-div {
+  margin: 0 2%;
+  vertical-align: middle;
+  flex: 1;
+  display: flex;
+  flex-direction: row;
+}
+
+.search-div form {
+  width: 100%;
+  margin: 0 0;
+  display: flex;
+  background: transparent;
+}
+
+.input1 {
+  position: relative;
+  border: 1px solid #d3d3d3;
+  width: 90%;
+  height: 30px;
+  margin: 0;
+  padding: 2px 6px;
+}
+
+.input2 {
```

Comment:
Your class name should be more descriptive, I noticed this is a button, why not call it `search-input`, `search-button` or any other descriptive name you want?